coverage.md

```diff
@@ -52,3 +52,4 @@ Manual Coverage
 | As a user, I can have a repository option that retains the latest N packages of the same name | PART | No coverage of packages with differing arch in same repo (src, i686, x86_64), no coverage of non-sync repo modifications, no coverage of modular RPMs being exempted. |
 | As a user, I can export RPM-repository content to be consumed by a downstream Pulp3 instance. | PART | basic pulp-export succeeds |
 | As a user, I can import RPM-repository content exported from an upstream Pulp3 instance. | PART | basic pulp-import succeeds |
+| As a user, I can export/import an RPM repository with a kickstart | PART | Need to use rpm-distribution-tree fixture and check content after import |
```

Comment:
What prevents to use rpm-distribution-tree fixture? The subrepos cleanup issue?

Reply:
That's correct. When you sync the new rpm-distribution-tree fixture into our test environment, tests try to clean up orphans between tests but this doesn't work since the kickstart orphans don't get cleaned up so it causes test failures throughout the pulp_rpm test suite. You can see the failures in https://github.com/pulp/pulp_rpm/pull/1832.